### PR TITLE
indenting and removing unnecessary few lines

### PR DESCRIPTION
**Pull request by hakanErgin**
- fixed console error for "body" being in "router"
- removed extra code giving not so important warnings
- indenting was seriously off. @ramzisalem you may want to check your text editor settings (prettier, eslint)
NOTE: realized that we are not using the images from upload directory (are hosted )

### CODE REVIEW

File: frontend/src/components/signUp.js [hunk 1]
```diff
@@ -1,49 +1,49 @@
 import React from 'react'
-import '../App.css'
+// import '../App.css'
```
Comment (hakanErgin) [inline frontend/src/components/signUp.js]: apparently we already have access to it here

File: frontend/src/components/homepage.js [hunk 2]
```diff
@@ -1,92 +1,91 @@
...
+  // Switch,
+  // Route,
+  // Link
+} from "react-router-dom";
 
-    return(
-        <Router>
+const Homepage = () => {
 
-        <body>
```
Comment (hakanErgin) [inline frontend/src/components/homepage.js]: if we put body in another container, we get errors from frontend console. 
image

Comment (hakanErgin) [reply frontend/src/components/homepage.js]: in this case 
`<Router><Body></Body></Router>`

**Comment by ramzisalem** [comment]
@hakanErgin thanks for checking the codes, I was yesterday night fixing these issue

**Comment by hakanErgin** [comment]
> @hakanErgin thanks for checking the codes, I was yesterday night fixing these issues

@ramzisalem I didn't know. we can cancel this PR if you want to push your changes. or you can merge if these are ok

**Comment by ramzisalem** [comment]
No go ahead with your changes, because I have to fix my editor firstly @hakanErgin

**Pull request merged**